pipelines/scheduler/luigi-scheduler.cfg

```diff
@@ -0,0 +1,10 @@
+[core]
+logging_conf_file=/etc/luigi/logging.cfg
+default-scheduler-port=8082
+
+[scheduler]
+record_task_history=true
+state-path=/var/lib/luigi-scheduler/state.pickle
+
+[task_history]
+db_connection=sqlite:////var/lib/luigi-scheduler/luigi-task-history.db
```

Comment:
Seems like it would be better to use the postgres database already set up for the project here.